gammapy/spectrum/spectrum_simulation.py

```diff
@@ -0,0 +1,85 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from gammapy.utils.random import get_random_state 
```

Comment:
Use relative imports consistently within Gammapy.
(we can discuss changing to absolute imports, it has advantages, but let's not mix.)